lstchain/calib/camera/r0.py

```diff
@@ -30,6 +30,10 @@ class CameraR0Calibrator(Component):
         Set to None if no Tool to pass.
     kwargs
     """
+    offset = Int(
+        300,
+        help='Define the offset of the baseline'
+    ).tag(config=True)
```

Comment:
this is the same offset as the one intended in `ctapipe_io_lst`, right?

Reply:
In principle yes, but this time the offset is set by Pawel's code and he use 300 as baseline.
Till now this is the active offset, for that the offset subtraction is done in this code for the moment